README.md

```diff
@@ -1,24 +1,157 @@
+**Blog App** ia a fully functional website that will show the list of posts and empower readers to interact with them by adding comments and liking posts.
 
-* Deployment instructions
+## 🛠 Built With <a name="built-with"></a>
 
-* ...
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Ruby on rails & Postgresql</summary>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **database is being set up in postgresql**
+- **data models are being created**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- To have a prior knowledge of Linters, Git, Ruby on Rails
+- Clone or fork the repository
+- A web browser (google chrome preferably)
+- A code editor (e.g vscode)
+
+### Setup
+
+Clone this repository to your desired folder:
+
+cd my-folder
+git clone https://github.com/Simpleshaikh1/blog-app
+
+### Install
+
+To install this project run the following commands in your terminal:
+
+- git clone https://github.com/Simpleshaikh1/blog-app
+- run bundle install
+
+### Usage
+
+To use this project run the following command:
+
+- run rails server
+
+### Run tests
+
+To run a test on this project run the following command:
+
+- ruby rspec/spec
+
+### Deployment
+
+You can deploy this project:
+
+- on any hosting services
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Deborah Fashoro**
+
+- GitHub: [@githubhandle](https://github.com/Simpleshaikh1)
+- Twitter: [@twitterhandle](https://twitter.com/simpleshaikh1)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/toyyib-abayomi)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Full blown blog app with proper navigation**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, kindly follow me on my social media handles listed above and leave me a star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank everyone who made this project possible.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] I think it would be better if you fix the broken `[MIT](./LICENSE)` link in your README file to make it more professional as it shows a **404** error page once clicked. And to fix that, you can only create a new file name `LICENSE` in your root directory and get its content from this **[template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)** or rename this `[MIT](./LICENSE)` to `[MIT](./MIT.md)`, then the created file in your directory name it `MIT.md` and use the same content in this **[template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)**
![image](https://github.com/Simpleshaikh1/blog-app/assets/83042742/9a961a66-dfaf-43cb-9382-dc44e4ee9723)